Add verifier name suffix for the duplicated template names (#62)
dl-solarity/hardhat-zkit: README.md

## Known limitations

- Currently there is minimal support for `var` Circom variables. Some circuits may not work if you are using complex `var`-dependent expressions.

> - Currentrly there is no support for arrays as parameters of the main component.

Reviewer: ```suggestion
- Also, passing arrays as parameters of a circuit `main` component is currently not supported.
```
